src/main.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { BrowserRouter as Router } from 'react-router-dom'
+
+import App from './App'
+
+import { LocaleContextProvider } from '@iteria-app/component-templates/src/i18n'
+
+// @ts-ignore
+if (!window.__skip_render) {
+  ReactDOM.render(
+    <LocaleContextProvider>
```

Comment:
please, try to move all providers to App.tsx

Reply:
Moving it to App.tsx causes this error:
![image](https://user-images.githubusercontent.com/31161371/148956779-4a163c93-3065-4058-a5b4-f3eb33166a87.png)